calcom/cal.com
# chore: app router - /booking pages

## What does this PR do?

- Fixes CAL-4893

### Tested

<img width="1512" alt="Screenshot 2024-12-18 at 2 05 24 PM" src="https://github.com/user-attachments/assets/edf6578f-393f-40cf-8f59-b4ef02297dba" />

<img width="1507" alt="Screenshot 2024-12-18 at 2 05 11 PM" src="https://github.com/user-attachments/assets/e7ba25a8-96af-469b-b4ef-875837c57728" />


## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- Go to https://dev-5rx6g6kri-cal-staging.vercel.app/bookings and press some bookings to check the booking pages
